portfolio/src/main/webapp/scripts/survey.js

```diff
@@ -0,0 +1,76 @@
+const subgenres = ['Body Horror', 'Slasher', 'Monster', 'Paranormal', 'Psychological Thriller'];
+const optionsContainer = document.getElementById('options-container');
+createSurvey();
+drawChart(); 
+
+/*
+ * Creates and adds individual option elements to the DOM
+ */ 
+function createSurvey() {
+  for (const optionName of subgenres) {
+    let option = document.createElement('input');
+    option.setAttribute('type', 'radio'); 
+    option.setAttribute('name', 'option');
+    option.setAttribute('value', optionName);
+    let optionLabel = document.createElement('label');
+    optionLabel.setAttribute('for', optionName);
+    optionLabel.textContent = optionName; 
+    optionsContainer.appendChild(option); 
+    optionsContainer.appendChild(optionLabel); 
+  }
+}
+
+/*
+ * Get the user's reponse to the survey
+ */  
+function getUserResponse() {
+  let options = document.getElementsByName('option'); 
+  for (const option of options) {
+    if (option.checked) {
+      addVote(option.value); 
```

Comment:
return after this since only option can be selected.

Reply:
I have added the return